mtags/src/main/scala-3/scala/meta/internal/pc/completions/CompletionsProvider.scala

```diff
@@ -257,46 +259,97 @@ class CompletionsProvider(
     val ident = completion.label
 
     val completionTextSuffix = completion.snippetSuffix.getOrElse("")
+
+    lazy val isInStringInterpolation =
+      path match
+        // s"My name is $name"
+        case (_: Ident) :: (_: SeqLiteral) :: (_: Typed) :: Apply(
+              Select(
+                Apply(
+                  Select(
+                    Select(
+                      _,
+                      name,
+                    ),
+                    _,
+                  ),
+                  _,
+                ),
+                _,
+              ),
+              _,
+            ) :: _ =>
```

Comment:
Can we accommodate these lines in a few lines? 😄

Reply:
Sure! Fixed.